README.md

```diff
@@ -1,6 +1,6 @@
-# CloudFuzz
+# CloudExec
```

Comment:
?!
Oh no, this must have gotten reverted during the super-squash. Thanks for noticing & cleaning this up so quickly 🙏